src/Blazored.Video/BlazoredVideo.razor.cs

```diff
@@ -39,11 +42,11 @@ public partial class BlazoredVideo
 		/// </summary>
 		[Parameter] public Dictionary<VideoEvents, VideoStateOptions> VideoEventOptions { get; set; }
 
-		private string UniqueKey = Guid.NewGuid().ToString("N");
+		protected string UniqueKey = Guid.NewGuid().ToString("N");
```

Comment:
What is the reason for changing these to protected?

Reply:
Simple reason that I was benefitting from it when I was overwriting your component and it was the most convinient way to access the Video element. I can understand that you dont want to make this private, but when its Protected someone who wants to extend the component could.